src/main/java/org/broadinstitute/hellbender/tools/spark/sv/utils/ExtractOriginalAlignmentRecordsByNameSpark.java

```diff
@@ -0,0 +1,86 @@
+package org.broadinstitute.hellbender.tools.spark.sv.utils;
+
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.apache.spark.api.java.JavaRDD;
+import org.apache.spark.api.java.JavaSparkContext;
+import org.apache.spark.broadcast.Broadcast;
+import org.broadinstitute.barclay.argparser.Argument;
+import org.broadinstitute.barclay.argparser.BetaFeature;
+import org.broadinstitute.barclay.argparser.CommandLineProgramProperties;
+import org.broadinstitute.hellbender.cmdline.StandardArgumentDefinitions;
+import org.broadinstitute.hellbender.cmdline.programgroups.StructuralVariationSparkProgramGroup;
+import org.broadinstitute.hellbender.engine.spark.GATKSparkTool;
+import org.broadinstitute.hellbender.exceptions.GATKException;
+import org.broadinstitute.hellbender.utils.gcs.BucketUtils;
+import org.broadinstitute.hellbender.utils.read.GATKRead;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.util.HashSet;
+
+/**
+ * Created by shuang on 3/3/17.
+ */
+@CommandLineProgramProperties(summary="Find original alignment records that have the requested read names and outputs them.",
+        oneLineSummary="Dump original alignment records that have the requested read names.",
+        usageExample = "ExtractOriginalAlignmentRecordsByNameSpark \\" +
+                "-I /path/to/my/dir/allAlignments.sam \\" +
+                "-O /path/to/my/dir/extractedOriginalAlignments.sam \\" +
+                "--readNameFile /path/to/my/dir/readNames.txt",
+        omitFromCommandLine = true,
+        programGroup = StructuralVariationSparkProgramGroup.class)
+@BetaFeature
+public final class ExtractOriginalAlignmentRecordsByNameSpark extends GATKSparkTool {
+    private static final long serialVersionUID = 1L;
+    private final Logger localLogger = LogManager.getLogger(ExtractOriginalAlignmentRecordsByNameSpark.class);
```

Comment:
I think that ```logger``` has already been initialized appropriately, you don't need to  declared another logger here unless I'm missing something.